x-pack/plugins/security_solution/cypress/integration/detection_rules/threshold_rule.spec.ts

```diff
@@ -174,7 +174,7 @@ describe('Detection rules, threshold', () => {
     cy.get(ALERT_GRID_CELL).contains(rule.name);
   });
 
-  it('Preview results of keyword using "host.name"', () => {
+  it.skip('Preview results of keyword using "host.name"', () => {
```

Comment:
Do you intend to fix these, or delete them?

Reply:
These will be on the list to actually fix after [this ticket](https://github.com/elastic/kibana/issues/120279) is completed